cmd/cosign/cli/errors.go

```diff
@@ -20,5 +20,5 @@ package cli
 type KeyParseError struct{}
 
 func (e *KeyParseError) Error() string {
-	return "either local key path (-key) or KMS path (-kms) must be provided, not both"
+	return "exactly one of: local key path (-key), KMS path (-kms) or hardware token (-piv-token) must be provided"
```

Comment:
should be `... key reference (-key), or hardware token (-piv-token) ...` after https://github.com/sigstore/cosign/pull/248